docs/4.x/system-check.md

```diff
@@ -0,0 +1,137 @@
+---
+category: Develop
+title: System check
+---
+# Enriching the system check
+
+Some plugins may only work when certain extra conditions are met, like having an unusual PHP extension installed or needing certain write permission on the file system. In these cases it is recommended to add new checks to the Matomo diagnostics system check which you can find by going to `Matomo Administration -> Diagnostic -> System Check`. 
+
+## Types of system checks 
```

Comment:
Given the code uses Diagnostic instead of "check", have you thought about calling them "diagnostics" here? As in `Types of diagnostics`

Reply:
Actually if the command is named `generate:system-check` then it doesn't matter, ignore this